jdbc/src/test/java/io/kestra/jdbc/repository/AbstractJdbcFlowRepositoryTest.java

```diff
@@ -35,18 +35,6 @@ public abstract class AbstractJdbcFlowRepositoryTest extends io.kestra.core.repo
     @Inject
     protected JooqDSLContextWrapper dslContextWrapper;
 
-    @Test
-    protected void find() {
```

Comment:
I don't understand why it's here, I believe it's an old code which should have been removed some time but it would have been useful not to have it as it would have spotted this bug (part of it at least)

Reply:
I don't understand why you remove that test

Reply:
OK, seen it, it override the test from the parent.
A pitty it didn't had the `@Override` annotation :(